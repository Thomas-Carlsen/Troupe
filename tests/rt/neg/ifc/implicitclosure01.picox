let val secret = mkSecret 42
    val f = if secret > 0 then fn x => x + 1
                          else fn x => x + 2
    val y = f 0
    val _ = adv y
in print y
end
