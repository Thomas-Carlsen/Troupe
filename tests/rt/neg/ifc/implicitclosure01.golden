2019-03-07T12:08:24.966Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mRuntime error in thread af4bd0d5-c223-495c-87e3-f44a4db61fee@{}%{}[39m
[31m>> Illegal flow in adv function:[39m
[31m |    pc: {}[39m
[31m | block: {#TOP}[39m
[31m | value: 1@{#TOP}%{#TOP}[39m
